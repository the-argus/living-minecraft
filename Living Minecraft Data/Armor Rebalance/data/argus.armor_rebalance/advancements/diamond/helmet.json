{
  "criteria": {
    "requirement": {
      "trigger": "minecraft:recipe_unlocked",
      "conditions": {
        "recipe": "minecraft:diamond_helmet"
      }
    }
  },
  "rewards": {
    "function": "argus.armor_rebalance:condition/diamond/helmet"
  }
}
