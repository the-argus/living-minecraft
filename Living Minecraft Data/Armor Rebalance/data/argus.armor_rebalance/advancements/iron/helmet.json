{
  "criteria": {
    "requirement": {
      "trigger": "minecraft:recipe_unlocked",
      "conditions": {
        "recipe": "minecraft:iron_helmet"
      }
    }
  },
  "rewards": {
    "function": "argus.armor_rebalance:condition/iron/helmet"
  }
}
