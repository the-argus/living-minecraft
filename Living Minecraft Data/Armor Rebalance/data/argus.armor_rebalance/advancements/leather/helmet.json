{
  "criteria": {
    "requirement": {
      "trigger": "minecraft:recipe_unlocked",
      "conditions": {
        "recipe": "minecraft:leather_helmet"
      }
    }
  },
  "rewards": {
    "function": "argus.armor_rebalance:condition/leather/helmet"
  }
}
