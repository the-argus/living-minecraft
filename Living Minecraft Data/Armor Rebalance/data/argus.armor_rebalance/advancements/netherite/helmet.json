{
  "criteria": {
    "requirement": {
      "trigger": "minecraft:recipe_unlocked",
      "conditions": {
        "recipe": "argus.armor_rebalance:netherite_helmet_shapeless"
      }
    }
  },
  "rewards": {
    "function": "argus.armor_rebalance:condition/netherite/helmet"
  }
}
