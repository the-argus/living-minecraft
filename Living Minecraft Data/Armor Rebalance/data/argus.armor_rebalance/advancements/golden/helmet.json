{
  "criteria": {
    "requirement": {
      "trigger": "minecraft:recipe_unlocked",
      "conditions": {
        "recipe": "minecraft:golden_helmet"
      }
    }
  },
  "rewards": {
    "function": "argus.armor_rebalance:condition/golden/helmet"
  }
}
